packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -24,9 +25,10 @@ import {
   CallErrorTarget,
   CallError
 } from './CallClientState';
-import { CallStateModifier } from './StatefulCallClient';
 
 enableMapSet();
+// Needed to generate state diff for verbose logging.
+enablePatches();
```

Comment:
is there any perf issues with this? (I see it was maybe removed from being enabled by default in immer since v6 - but couldn't see why that was
![image](https://user-images.githubusercontent.com/2684369/153241392-86ed4ca4-ce82-4b73-bf34-9c8b4c8db4c8.png)
)

Reply:
No performance implication. The implication is in bundle size, but it's small.

Reply:
cf: 

https://immerjs.github.io/immer/installation#pick-your-immer-version

![image](https://user-images.githubusercontent.com/82062616/153259032-84aaae60-5dd1-44bd-b00b-5dc25dd9a801.png)

Reply:
thanks!